### PR TITLE
Ping non root

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17103

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

## Related PR
#3673

## Required version of Demisto
5.0

## Does it break backward compatibility?
   - No: we update docker image only in 5.0

## Must have
- [x] Tests

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Integrations/ImageOCR/Pipfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "12fda029f003654818dd80d8c29a8db62d03e03a080818d54f097d6d82991f56"
+            "sha256": "2197a28c3a643c22111a754c6a2a77781af6e24cc896194424a14c61711c72e7"
```
Comment (anara123) [inline Integrations/ImageOCR/Pipfile.lock]: you want to include ImageOCR Pipfile changes too? Is it on purpose here?

File: Scripts/Ping/Ping.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        s = re.search(r"rtt min/avg/max/mdev = (.+)/(.+)/(.+)/(.+)\s+ms", ping_out)
+        if not s:
+            raise ValueError("Couldn't parse ping statistics:\n" + ping_out)
+        res['ret_code'] = 0
+        res['destination'] = dest
+        res['min_rtt'] = s.group(1)
+        res['avg_rtt'] = s.group(2)
+        res['max_rtt'] = s.group(3)
+        res['mdev_rtt'] = s.group(4)
+        demisto.results({
```
Comment (anara123) [inline Scripts/Ping/Ping.py]: use `return_outputs` instead `demisto.results`
`return_outputs(readable_outputs=tableToMarkdown("Ping Results", res), outputs={"Ping": res},raw_response=res)`

File: Integrations/ImageOCR/Pipfile.lock (hunk 1 above)
Comment (glicht) [reply Integrations/ImageOCR/Pipfile.lock]: Yes. It's ok. I had there an un-needed dependency for some reason.

File: Scripts/Ping/Ping.py (hunk 2 above)
Comment (glicht) [reply Scripts/Ping/Ping.py]: done

**Comment by glicht** [comment]
@anara123 fixed the destination_ip output

**Pull request merged**